app/src/app/(auth)/sign-up.tsx

```diff
@@ -1,80 +0,0 @@
-import { View, Text, TextInput, StyleSheet, Alert } from 'react-native';
```

Comment:
this page seems unnecessary since i already thought the link would send the request for me. it was also weird having to re-enter my info that was already on the previous page